apps/screenshot-diff/src/util/image-diff/index.ts

```diff
@@ -23,7 +23,7 @@ export const diffImages = async ({
   });
 
   return {
-    score: difference.value,
+    score: difference.value < 0.00005 ? 0 : difference.value,
```

Comment:
Why? Thresold should be sufficient. For big images 1% is enormous, for a small one it is just a pixel. So it makes no sense to have a check here.

Reply:
A agree that percentage is not the best indicator. Nonetheless, the [border test](apps/screenshot-diff/src/util/image-diff/index.test.ts) return a score around "0.00001272". Before odiff, we ignored this kind of diff. What should we do now?

Reply:
OK